README.md

```diff
@@ -596,6 +596,8 @@ _Related: [Conference Management](#conference-management)_
 - [Jitsi Meet](https://jitsi.org/Projects/JitsiMeet) - Jitsi Meet is an OpenSource (MIT) WebRTC Javascript application that uses Jitsi Videobridge to provide high quality, scalable video conferences. ([Source Code](https://github.com/jitsi/jitsi-meet)) `MIT` `Javascript`
 - [Jitsi Video Bridge](https://jitsi.org/Projects/JitsiVideobridge) - WebRTC compatible Selective Forwarding Unit (SFU) that allows for multiuser video communication. ([Source Code](https://github.com/jitsi/jitsi-videobridge)) `Apache-2.0` `Java`
 - [LiveKit](https://livekit.io/) - Modern, scalable WebRTC conferencing platform with client SDKs. ([Demo](https://livekit.io/playground), [Source Code](https://github.com/livekit/livekit-server)) `Apache-2.0` `Go`
+- [MiroTalk P2P](https://p2p.mirotalk.com) - WebRTC P2P - simple, secure, fast real-time video conferences compatible with all browsers and platforms. ([Demo](https://p2p.mirotalk.com/newcall), [Source Code](https://github.com/miroslavpejic85/mirotalk)) `AGPL-3.0` `JavaScript`
+- [MiroTalk SFU](https://sfu.mirotalk.com) - WebRTC SFU - simple, secure, scalable real-time video conferences compatible with all browsers and platforms. ([Demo](https://sfu.mirotalk.com/newroom), [Source Code](https://github.com/miroslavpejic85/mirotalksfu)) `AGPL-3.0` `JavaScript`
```

Comment:
The description should say what the software does. 
- Please change the description to `Simple, secure, scalable real-time video conferences up to 4k, compatible with all browsers and platforms.`

> https://github.com/miroslavpejic85/mirotalksfu/blob/main/docs/self-hosting.md
- Please change the language tag to `Docker/Nodejs`